src/actions/userActions.js

```diff
@@ -2,16 +2,8 @@ import { GoogleSignin } from '@react-native-community/google-signin';
 import { createThunk, createAction } from '@rootstrap/redux-tools';
 
 import userService from 'services/userService';
-import parseError from 'utils/parseError';
 
-export const login = createThunk('LOGIN', async token => {
-  try {
-    const { data } = await userService.login({ token });
-    return data.user;
-  } catch ({ response }) {
-    throw parseError(response);
-  }
-});
+export const login = createThunk('LOGIN', async token => userService.login({ token }).data);
```

Comment:
Does this work?
I think it should be
```suggestion
export const login = createThunk('LOGIN', async token => (await userService.login({ token })).data);
```

I can't think of a use case in which an async function does not have an await inside...

Reply:
If I do it like that I get a warning because there is nothing that needs to be executed inside the action after the wait